contrib/cf-remote/cf_remote/main.py

```diff
@@ -56,41 +96,48 @@ def run_command_with_args(command, args):
             call_collect=args.call_collect,
             edition=args.edition)
     elif command == "packages":
-        commands.packages(tags=args.args, version=args.version, edition=args.edition)
+        commands.packages(tags=args.tags, version=args.version, edition=args.edition)
     elif command == "run":
         commands.run(hosts=args.hosts, command=" ".join(args.args))
     elif command == "sudo":
         commands.sudo(hosts=args.hosts, command=" ".join(args.args))
     elif command == "scp":
         commands.scp(hosts=args.hosts, files=args.args)
+    elif command == "spawn":
+        if args.list_platforms:
+            commands.list_platforms()
+            return
+        elif args.init_config:
+            commands.init_cloud_config()
+            return
+        # else
+        if args.role.endswith("s"):
+            # role should be singular
+            args.role = args.role[:-1]
+        return commands.spawn(args.platform, args.count, args.role, args.name)
```

Comment:
As pointed out by LGTM it's a little weird that you return a value here, when all the other branches don't return like this.

Reply:
Yeah, I admit it's a weird mixture. But it was an attempt to seed a better future where *cf-remote* will actually exit with a status code returned by the command it runs. Removed for now.